mockserver/nodeGroup/nodeGroup.manager.js

```diff
@@ -6,7 +6,10 @@ import Role from '../user/enum.js';
 class NodeGroupManager {
   static async getNodeGroups(req, res) {
     const requester = jwt.decode(req.headers.authorization.split(' ')[1]);
-    const unitNodesGroup = nodeGroups.filter((nodeGroup) => nodeGroup.unitId === requester.unitId);
+    const { unitId } = req.query;
+    const unitNodesGroup = unitId
+    ? nodeGroups.filter((nodeGroup) => nodeGroup.unitId === unitId)
+    : nodeGroups.filter((nodeGroup) => nodeGroup.unitId === requester.unitId);
```

Comment:
Remove redundant code:

```suggestion
    const unitNodesGroup = nodeGroups.filter((nodeGroup) => nodeGroup.unitId === (unitId ?? requester.unitId) );
```

Reply:
it is mock. who cares?